lib/Elastica/Bulk/ResponseSet.php

```diff
@@ -42,16 +42,13 @@ public function getBulkResponses()
      */
     public function getError()
     {
-        $error = '';
-
         foreach ($this->getBulkResponses() as $bulkResponse) {
             if ($bulkResponse->hasError()) {
-                $error = $bulkResponse->getError();
-                break;
+                return $bulkResponse->getError();
```

Comment:
@ReenExe Like I mentioned before, the changes are not wrong at all and lots of projects do it this way. It is only about my personal preference. Unfortunately you are right and it is probably not followed in all parts of the code :-(